### PR TITLE
zh-CN: translate cargo

### PR DESCRIPTION
**Pull request by mgeisler**
Part of #324.

### CODE REVIEW

File: po/zh-CN.po [hunk 1]
```diff
@@ -1732,10 +1764,43 @@ msgid ""
 "[cargo clippy]).\n"
 "  * Read more from the [official Cargo Book]"
 msgstr ""
+"* Rust 有一个快速发布时间表，每六周就会发布一次\n"
+"新版本。新版本保持与\n"
+"旧版本的向后兼容性，还实现了新功能。\n"
```
Comment (NoahDragon) [inline po/zh-CN.po:1769]:
```suggestion
"旧版本的向后兼容性，还添加了新功能。\n"
```

“添加” means adding, which would be more fit here comparing “实现” (implement).

File: po/zh-CN.po [hunk 2]
```diff
@@ -1767,19 +1838,28 @@ msgid ""
...
+"```"
 
 #: src/cargo/code-samples.md:19
 msgid ""
 "You can use <kbd>Ctrl + Enter</kbd> to execute the code when focus is in "
 "the\n"
 "text box."
 msgstr ""
+"当焦点位于\n"
+"文本框时，你可以使用 <kbd>Ctrl + Enter</kbd> 来执行代码。"
```
Comment (NoahDragon) [inline po/zh-CN.po:1854]:
```suggestion
"当文本框为\n"
"焦点时，你可以使用 <kbd>Ctrl + Enter</kbd> 来执行代码。"
```
nit: change the order.

File: po/zh-CN.po [hunk 3]
```diff
@@ -17145,7 +17295,7 @@ msgstr ""
 
 #: src/credits.md:1
 msgid "# Credits"
-msgstr ""
+msgstr "# 鸣谢"
```
Comment (NoahDragon) [inline po/zh-CN.po:17298]: "credits" has translated in #867

**Review by NoahDragon — APPROVED** [review]

File: po/zh-CN.po (hunk 3 above)
Comment (mgeisler) [reply po/zh-CN.po:17298]: Strange, I wonder how the translation got copied into here. At least it doesn't conflict :smile:

**Pull request merged**